src/pages/LogIN/LogInPage.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import "../../components/ProductCard/Productcard.css";
+import { FaEyeSlash } from "react-icons/fa";
+import { Link } from 'react-router-dom';
+import "../LogIN/LogInPage.css";
+
+ export const LogInPage = () => {
```

Comment:
some minor changes CSS can be written to center this component to page for all sizes

Reply:
if i gave it parent div to center property it goes above the header that's why i gave a margin-left